extensions/binding/org.eclipse.smarthome.binding.bluetooth/src/main/java/org/eclipse/smarthome/binding/bluetooth/notification/BluetoothScanNotification.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.binding.bluetooth.notification;
+
+/**
+ * The {@link BluetoothScanNotification} provides a notification of a received scan packet
+ *
+ * @author Chris Jackson - Initial contribution
+ */
+public class BluetoothScanNotification extends BluetoothNotification {
+    /**
+     * The receive signal strength for this beacon packet
+     */
+    private int rssi = Integer.MIN_VALUE;
+
+    /**
+     * The raw data
+     */
+    private byte[] data = null;
+
+    /**
+     * The manufacturer specific data
+     */
+    private byte[] manufacturerData = null;
+
+    /**
+     * The beacon type
+     */
+    private BluetoothBeaconType beaconType = BluetoothBeaconType.BEACON_UNKNOWN;
+
+    /**
+     * The device name
+     */
+    private String name = new String();
```

Comment:
Explicit String instance creation and hence the object instance cannot be interned. Assigning to empty value should suffice.